### ISSUE TITLE
Filling custom fields throws an error

### ISSUE THREAD

**Issue by mrimann** [Icinga/icingaweb2-module-jira#35]
I've freshly installed the module and tested it to work basically.

We use "Resource-Pools" to target tasks to. So instead of directly assigning a task to a specific person, I assign it to the appropriate "Pool" and the members of that team then pull those tasks and work through them. One of the Pools is our "IT Operations Pool" and I'd like to directly send new tickets from Icinga into that pool.

Therefore I followed the docs https://github.com/Icinga/icingaweb2-module-jira/blob/master/doc/03-Configuration.md, especially the section about "Fill JIRA Custom Fields". My template looks like this now:

```
[icinga-workflow]
customfield_10200 = "10300"
```

Customfield "10200" is the "Pool" field. In the JIRA GUI that's a dropdown with the list of allowed values. I've tried the textual representation "Ops" and the internal ID of that dropdown-value - both didn't work:

When I navigate through the Icingaweb-GUI to a service and from there create a JIRA issue **without template, it works**.
But when I do the same and choose the above showed template, I get the following error shown:

> Konnte keine gültige 'ID' bzw. keinen gültigen 'Wert' im Objekt "Übergeordnete Option" finden.

![image](https://user-images.githubusercontent.com/1093784/69485603-e7a16a80-0e41-11ea-996f-9bf626287588.png)

I guess that's somehow coming from JIRA since Icingaweb is completely in english - but our JIRA instance is running in German.

**Comment by Thomas-Gelf** [comment]
@mrimann: could you please try to set `Pool.value = "10300"` instead of `customfield_10200 = "10300"`?

**Comment by mrimann** [comment]
I tried with the real name "Nötige Ressourcen" of that field like this in the config:
```
[icinga-workflow]
"Nötige Ressourcen" = "10300"
```

Now as soon as I go to a Service > JIRA Tasks > Create, I get the following error shown (instead of the expected form):
ERROR: Couldn't parse the INI file `/etc/icingaweb2/modules/jira/templates.ini'

![image](https://user-images.githubusercontent.com/1093784/69709085-08491900-10fd-11ea-913f-77c2eb63544b.png)

I cannot simply change the Field in JIRA - so I guess it's either the space or the umlaut in the field-name that breaks the parsing.

After asking Google on how to do encode umlauts etc. in *.ini files, I fould the "" to be forbidden in the key. So I tried it without:
```
[icinga-workflow]
Nötige Ressourcen = "10300"
```

Now the form is shown - and when I chose this template + submit it, I get the same error again as before:

Konnte keine gültige 'ID' bzw. keinen gültigen 'Wert' im Objekt "Übergeordnete Option" finden.
![image](https://user-images.githubusercontent.com/1093784/69709744-4a268f00-10fe-11ea-9343-6844df87ad05.png)

After re-reading your suggestion, I've changed `Nötige Ressourcen` to `Nötige Resourcen.value` and with the following config tested again:
```
[icinga-workflow]
Nötige Ressourcen.value = Hosting
```

This seems to work as I need it - thanks @Thomas-Gelf !
